lighthouse-core/report/html/report-styles.css

```diff
@@ -387,14 +386,20 @@
   margin-right: 0;
 }
 
-.lh-audit__title {
+.lh-audit__display-text {
   flex: 1;
 }
 
+/* Prepend display text with separator. But not in Opportunities. */
+.lh-audit-group:not(.lh-audit-group--load-opportunities) .lh-audit__display-text:not(:empty):before {
```

Comment:
while this selector does look crazy, it turns out to be reasonable..  👍

Reply:
:)